dotcom-rendering/src/web/components/SignInGate/gateDesigns/SignInGateCopyTestJan2023.tsx

```diff
@@ -0,0 +1,153 @@
+import { cmp } from '@guardian/consent-management-platform';
+import { Button, Link, LinkButton } from '@guardian/source-react-components';
+import { trackLink } from '../componentEventTracking';
+import type { SignInGateProps } from '../types';
+import {
+	actionButtons,
+	bodyBold,
+	bodySeparator,
+	bodyText,
+	faq,
+	firstParagraphOverlay,
+	headingStyles,
+	hideElementsCss,
+	laterButton,
+	privacyLink,
+	registerButton,
+	signInGateContainer,
+	signInHeader,
+	signInLink,
+} from './shared';
+
+export const SignInGateCopyTestJan2023 = ({
+	signInUrl,
+	guUrl,
+	dismissGate,
+	abTest,
+	ophanComponentId,
+	isMandatory = false,
+}: SignInGateProps) => {
+	const getCopyTextFromVariant: (abVariant: string | undefined) => string = (
+		abVariant,
+	) => {
+		switch (abVariant) {
+			case 'sign-in-gate-copy-1': {
+				return "Register: it's quick and easy";
+			}
+			case 'sign-in-gate-copy-2': {
+				return 'Take a moment to register';
+			}
+			default: {
+				return "Register: it's quick and easy";
```

Comment:
I would have this be the control. So the current copy. I'm pretty sure it will never be hit but clearer that way.

Reply:
Yep good idea, added